support/src/indirect_intrusive_heap.h

```diff
@@ -48,7 +48,7 @@ namespace crimson {
 	   typename T,
 	   IndIntruHeapData T::*heap_info,
 	   typename C,
-	   uint K = 2>
+	   unsigned int K = 2>
```

Comment:
nit, might want to just put `unsigned` instead of `unsigned int` for the sake of consistency.